tests/commons/glob/test_glob.py

```
@@ -0,0 +1,100 @@
+
+
+def test_visitGlobPaths():
+    assert visitGlobPaths
+    assert GlobMatchVisitor
+
+    class TestVisitor(GlobMatchVisitor):
+        def __init__(self):
+            self.visited = []
+
+        def visitExcluded(self, path: Path) -> None:
+            raise Exception(f"in this test we should not exclude path {path}")
```

Ruff (0.8.2)

Create your own exception

(TRY002)